pandas/tests/test_groupby.py

```diff
@@ -3769,6 +3769,21 @@ def test_getitem_list_of_columns(self):
         assert_frame_equal(result2, expected)
         assert_frame_equal(result3, expected)
 
+    def test_getitem_numeric_column_names(self):
+        df = DataFrame({0: list('abcd') * 2,
```

Comment:
Can you add the issue number (or in this case PR number) as a comment on the first line?